lib/tools/adb-emu-commands.js

```diff
@@ -5,42 +5,46 @@ import path from 'path';
 
 let emuMethods = {};
 
-emuMethods.isEmulatorConnected = async function (udid) {
-  let emulators = await this.getConnectedEmulators();
-  if (!emulators.length) {
-    return false;
+emuMethods.getEmulators = async function() {
```

Comment:
`memoize` caches forever, `debounce` caches for a certain amount of time.

@vrunoa i guess we have to make an estimate of the likelihood that emulator state will change during the lifetime of an ADB instance.

debouncing to like 5s might be a good idea just to avoid making lots of calls at the same time? you're better equipped to judge than i am.